projects/components/src/table/cells/table-cell-parser-base.ts

```diff
@@ -1,9 +1,12 @@
+import { Injector } from '@angular/core';
 import { TableRow } from '../table-api';
 import { CoreTableCellParserType } from './types/core-table-cell-parser-type';
 
 export abstract class TableCellParserBase<TCellData, TValue, TFilterValue> {
   public static readonly type: CoreTableCellParserType;
 
+  public constructor(protected readonly injector: Injector) {}
```

Comment:
Yeah, this is annoying.

The only place we need this right now is in `BackendIconTableCellParser` because it needs to inject `EntityIconLookupService`.

Reply:
Oh this is because we treat it as the base class and construct it. Will finish this thought in that thread.